### ISSUE TITLE
Hard to follow both the slides and the commands in `batch_single_subject.sh`

### ISSUE THREAD

**Issue by joshuacwnewton** [spinalcordtoolbox/sct_tutorial_data#7]
Each of the commands in `batch_single_subject.sh` is featured in our SCT Course slides. However, mid-course, some people might be referencing the slides (e.g. slide number ##, etc.) while some people might be referencing the script (line number ##, etc.)

It would be nice if there was a way to link the two somehow, so that people can navigate between the two more easily. Ideas:

- Manually add slide numbers to the script, or script numbers to the slides. 
   - (Might be hard to maintain... adding a new command/page will offset all of the numbers and force us to update them)
- Add section numbers 
   - This might work better, because our slide sections are already numbered, and section numbers are less sensitive to changes.

**Comment by jcohenadad** [comment]
> Add section numbers

I like that option. Especially because we are _kind of_ currently doing it (ie: the script is already organized into sections).

**Comment by joshuacwnewton** [comment]
I'm going to close this issue just to centralize discussion in Issue 2407 (the SCT issue about copying and pasting during the course). 

I have a lot of thoughts on the matter, and will be making a big summary of all of the things we've tried so far, that way it will save people some effort in preparation of our next SCT meeting (where there is an action item for everyone).
